thistlethwaiteAlgo.py

```diff
@@ -4,6 +4,17 @@
 import sys
 import time
 
+# COLORS
```

Comment:
Maybe we should also use these colors in `RubikState`

Reply:
Okay ✅